scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -164,6 +164,68 @@ function doStartEclipse() {
   fi
 }
 
+function doStartEclipseBlocking() {
+  if doIsWindows
+  then
+    # shellcheck disable=SC2086
+    "eclipse" /B "${ECLIPSE_HOME}/eclipsec" -clean -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${JAVA_HOME}/bin/javaw" -showlocation "${WORKSPACE}" -vmargs ${ECLIPSE_VMARGS}
+  else
+    # shellcheck disable=SC2086
+    "${ECLIPSE_HOME}/eclipse" -clean -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${JAVA_HOME}/bin/java" -showlocation "${DEVON_IDE_HOME/*\//}/${WORKSPACE}" -vmargs ${ECLIPSE_VMARGS}
+  fi
+}
+
+# $1: project type
+# $2: optional project path
+function doImportProject() {
+  local setup_ws="${WORKSPACE_PATH}/.metadata/.plugins/org.eclipse.oomph.setup/workspace.setup"
+  local setup_templ="${DEVON_IDE_HOME}/scripts/templates/oomph"
+  local root_path="${OLDPWD}"
```

Comment:
I don't understand, why you set ${OLDPWD} as the default for `${root_path}` here, which is used, when the optional second parameter for this function is not given. Why should the directory before the last `cd` command a valid default?
Also as mentioned above, I wonder, if not "." were a more useful default. (Only if relative path names work)